packages/recorder/plugins/amplitude/amplitude.js

```diff
@@ -0,0 +1,209 @@
+const AudioContext =
+  window.AudioContext || /* istanbul ignore next */ window.webkitAudioContext;
+let audioContext;
+
+/**
+ * Analyze the amplitude of sound with this object.
+ *
+ * Amplitude is the magnitude of vibration. Sound is vibration, so its amplitude is closely related
+ * to volume/loudness.
+ *
+ * The original amplitude values for digital audio are between -1.0 and 1.0. But the RMS will always
+ * be positive, because it is squared. And, rather than using instantaneous amplitude readings that
+ * are sampled at a rate of 44,100 times per second, the RMS is an average over time which better
+ * represents how we hear amplitude.
```

Comment:
and perhaps also less sampling because of performance ;-)

Reply:
There is no downsampling on the recorder, I don't think it is relevant for this doc-block?